debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgisGeometry.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.connector.postgresql;
+
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+
+
+public class PostgisGeometry {
+
+    private final byte[] wkb;
+    private final int srid;
+
+    /**
+     * Create a PostgisGeometry using the supplied PostGIS Hex EWKB string.
+     * SRID is extracted from the EWKB
+     */
+    public static PostgisGeometry fromHexEWKB(String hexEWKB) {
```

Comment:
What is EWKB?

In terms of element names, only the first letter of an acronym should be upper-cased: `fromHexEwkb()` (same below).

Reply:
Extended-well-known-binary. It's a compatible extension of the standard WKB used by PostGIS which encodes the SRID into the binary.

So all EWKB is WKB, and all WKB is valid EWKB (with no SRID)